src/wasm/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../node_modules/assemblyscript/std/assembly.json",
```

Comment:
About this, I have questions:

 1.  What does this file do? (Sorry, I am still only getting started with AssemblyScript.)
 2.  Is the `extends` field necessary? It does not seem to affect the output of `asc`.
 3.  Does the `include` field mean that `asc` will compile all `.ts` files? `buildwasm.js` currently passes the path to the source file to `asc` directly.

Reply:
It's actually matter if someone work in VS Code or Atom editors or/and use tslint. `assembly.json` contain setup and definitions for AS and TS (if someone wont compile via tsc same code)

Reply:
Thanks for explaining!